_benchmark/configuring-benchmark.md

```diff
@@ -0,0 +1,202 @@
+| `cache.days` | Integer | The number of days for which an artifact should be kept in the source artifact cache. Default is `7`. |
+
+## benchmarks
+
+This section contains the settings that can be customized in the OpenSearch Benchmark data directory.
+
+| Parameter | Type | Description |
+| :---- | :---- | :---- |
+| `local.dataset.cache` | String | The directory in which benchmark data sets are stored. Depending on the benchmarks that are run, this directory may contain hundreds of GB of data. |
+
+## results_publishing
+
+This section defines how benchmark metrics are stored.
+
+| Parameter | Type | Description |
+| :---- | :---- | :---- |
+| `datastore.type` | String | If set to `in-memory` all metrics will be kept in memory while running the benchmark. If set to `opensearch` all metrics will instead be written to a persistent metrics store and the data
+are available for further analysis. Default is `in-memory`. |
+| `sample.queue.size` | Function | The number of metrics samples that can be stored in OpenSearch Benchmark’s in-memory queue. Default is `2^20`. | 
+| metrics.request.downsample.factor | Integer| (default: 1): Determines how many service time and latency samples should be kept in the metrics store. By default all values will be kept. If you want to, for example. keep only every 100th sample, specify `100`. This is useful to avoid overwhelming the metrics store in benchmarks with many clients. Default is `1`. |
+| `output.processingtime` | Boolean | If set to `true`, OpenSearch will show the additional metric processing time in the command line report. Default is `false`. |
+
+### `datastore.type` parameters
+
+When `datastore.type` is set to `opensearch`, the following reporting settings can be customized.
+
+| Parameter | Type | Description |
+| :---- | :---- | :---- |
+| `datastore.host` | IP address | The host name of the metrics store, for example `124.340.200.22`. |
+| datastore.port| Port | The port number of the metrics store, for example, `9200`. |
+| `datastore.secure` | Boolean | If set to `false`, OpenSearch assumes a HTTP connection. If set to true, it assumes a HTTPS connection. |
+| `datastore.ssl.verification_mode` | String | When set to the default `full`, the metric store’s SSL certificate is checked. To disable certificate verification set this value to `none`. |
+| `datastore.ssl.certificate_authorities` | String | Determines the path on the local file system to the certificate authority’s signing certificate.
+| `datastore.user` | Username |  Sets the name of the OpenSearch user for the metrics store. |
+| `datastore.password`: | String | Sets the password of the OpenSearch user for the metrics store. Alternatively, this password can be configured using the `OPENSEARCH_REPORTING_DATASTORE_PASSWORD` environment variable, which avoids storing credentials in a plain text file. The environment variable will take precedence over the config file if both define a password. |
```

Comment:
We can swap `OPENSEARCH_REPORTING_DATASTORE_PASSWORD` for `OSB_RESULTS_PUBLISHING_PASSWORD` to stay in line with "Results_Publishing" section.